### PR TITLE
Support netstandard2.0

### PR DESCRIPTION
**Pull request by baruchiro**
For #43

### CODE REVIEW

File: src/DocoptNet/DocoptBaseException.cs [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ﻿using System;
+#if NET40
```
Comment (iwillspeak) [inline src/DocoptNet/DocoptBaseException.cs]: prefer NETFRAMEWORK for checking this I think.

File: src/DocoptNet/DocoptNet.csproj [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
-    <TargetFrameworks>net40;netstandard1.5</TargetFrameworks>
+    <TargetFrameworks>net40;netstandard1.5;netstandard2.0</TargetFrameworks>
     <AssemblyName>DocoptNet</AssemblyName>
     <Version>0.6.1.10</Version>
     <AssemblyOriginatorKeyFile>DocoptNet.snk</AssemblyOriginatorKeyFile>
     <SignAssembly>true</SignAssembly>
     <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
     <PackageId>DocoptNet</PackageId>
     <Copyright>Copyright (c) 2013 Dinh Doan Van Bien, dinh@doanvanbien.com</Copyright>
-    <NetStandardImplicitPackageVersion Condition=" '$(TargetFramework)' == 'netstandard1.5' ">1.6.0</NetStandardImplicitPackageVersion>
```
Comment (iwillspeak) [inline src/DocoptNet/DocoptNet.csproj]: If we aren’t referencing a newer version of the .net standard what is gained by this?

**Comment by baruchiro** [comment]
Ohhh, Just a few days before it turns one year old.

**Comment by voieducode** [comment]
I know. I am trying to find some time to update to net5...

**Pull request merged**